Error: The terraform-provider-azurerm_v2.68.0_x5 plugin crashed!

<!---
Please note the following potential times when an issue might be in Terraform core:

* [Configuration Language](https://www.terraform.io/docs/configuration/index.html) or resource ordering issues
* [State](https://www.terraform.io/docs/state/index.html) and [State Backend](https://www.terraform.io/docs/backends/index.html) issues
* [Provisioner](https://www.terraform.io/docs/provisioners/index.html) issues
* [Registry](https://registry.terraform.io/) issues
* Spans resources across multiple providers

If you are running into one of these scenarios, we recommend opening an issue in the [Terraform core repository](https://github.com/hashicorp/terraform/) instead.
--->

<!--- Please keep this note for the community --->

### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

<!--- Thank you for keeping this note for the community --->

### Terraform (and AzureRM Provider) Version

<!--- Please run `terraform -v` to show the Terraform core version and provider version(s). If you are not running the latest version of Terraform or the provider, please upgrade because your issue may have already been fixed. [Terraform documentation on provider versioning](https://www.terraform.io/docs/configuration/providers.html#provider-versions). --->

### Affected Resource(s)

<!--- Please list the affected resources and data sources. --->

* `azurerm_logic_app_workflow`

### Terraform Configuration Files
NOTE: The x characters denote exact case and length of resource name and actual name in Azure

<!--- Information about code formatting: https://help.github.com/articles/basic-writing-and-formatting-syntax/#quoting-code --->

```hcl
resource "azurerm_logic_app_workflow" "xxxxxxxxxxxx-xxxxxx-xxxxxxxxx-xxxxxxxxx" {
  name                = "Xxxxxxxxxxxx-Xxxxxx_Xxxxxxxxx_Xxxxxxxxx"
  location            = var.location
  resource_group_name = azurerm_resource_group.xxx-xxx-xxxxxx.name
  workflow_schema     = "https://schema.management.azure.com/providers/Microsoft.Logic/schemas/2016-06-01/workflowdefinition.json#"
}
```

### Panic Output

╷
│ Error: Plugin did not respond
│ 
│ The plugin encountered an error, and failed to respond to the plugin.(*GRPCProvider).ReadResource call. The plugin logs may contain more details.
╵


Stack trace from the terraform-provider-azurerm_v2.68.0_x5 plugin:

panic: interface conversion: interface {} is nil, not string

goroutine 46 [running]:
github.com/terraform-providers/terraform-provider-azurerm/azurerm/internal/services/logic.resourceLogicAppWorkflowRead(0xc00011f180, 0x62999e0, 0xc000e70a80, 0x0, 0x0)
        /opt/teamcity-agent/work/5d79fe75d4460a2f/src/github.com/terraform-providers/terraform-provider-azurerm/azurerm/internal/services/logic/logic_app_workflow_resource.go:310 +0xf71
github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema.(*Resource).read(0xc000e1a0e0, 0x71ffb08, 0xc001b40180, 0xc00011f180, 0x62999e0, 0xc000e70a80, 0x0, 0x0, 0x0)
        /opt/teamcity-agent/work/5d79fe75d4460a2f/src/github.com/terraform-providers/terraform-provider-azurerm/vendor/github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema/resource.go:335 +0x1ee
github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema.(*Resource).RefreshWithoutUpgrade(0xc000e1a0e0, 0x71ffb08, 0xc001b40180, 0xc0007dc460, 0x62999e0, 0xc000e70a80, 0xc00011a710, 0x0, 0x0, 0x0)
        /opt/teamcity-agent/work/5d79fe75d4460a2f/src/github.com/terraform-providers/terraform-provider-azurerm/vendor/github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema/resource.go:624 +0x1cb
github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema.(*GRPCProviderServer).ReadResource(0xc000722210, 0x71ffb08, 0xc001b40180, 0xc001b401c0, 0xc001b40180, 0x100b705, 0x65ad180)
        /opt/teamcity-agent/work/5d79fe75d4460a2f/src/github.com/terraform-providers/terraform-provider-azurerm/vendor/github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema/grpc_provider.go:575 +0x43b
github.com/hashicorp/terraform-plugin-go/tfprotov5/server.(*server).ReadResource(0xc001cb1900, 0x71ffbb0, 0xc001b40180, 0xc001b460c0, 0xc001cb1900, 0xc001bbe1e0, 0xc0010f1ba0)
        /opt/teamcity-agent/work/5d79fe75d4460a2f/src/github.com/terraform-providers/terraform-provider-azurerm/vendor/github.com/hashicorp/terraform-plugin-go/tfprotov5/server/server.go:298 +0x105
github.com/hashicorp/terraform-plugin-go/tfprotov5/internal/tfplugin5._Provider_ReadResource_Handler(0x6768140, 0xc001cb1900, 0x71ffbb0, 0xc001bbe1e0, 0xc001b46060, 0x0, 0x71ffbb0, 0xc001bbe1e0, 0xc001b48600, 0x2c4)
        /opt/teamcity-agent/work/5d79fe75d4460a2f/src/github.com/terraform-providers/terraform-provider-azurerm/vendor/github.com/hashicorp/terraform-plugin-go/tfprotov5/internal/tfplugin5/tfplugin5_grpc.pb.go:344 +0x214
google.golang.org/grpc.(*Server).processUnaryRPC(0xc00022a8c0, 0x724a198, 0xc000299500, 0xc000dc06c0, 0xc0013c0e10, 0xa764d30, 0x0, 0x0, 0x0)
        /opt/teamcity-agent/work/5d79fe75d4460a2f/src/github.com/terraform-providers/terraform-provider-azurerm/vendor/google.golang.org/grpc/server.go:1292 +0x52b
google.golang.org/grpc.(*Server).handleStream(0xc00022a8c0, 0x724a198, 0xc000299500, 0xc000dc06c0, 0x0)
        /opt/teamcity-agent/work/5d79fe75d4460a2f/src/github.com/terraform-providers/terraform-provider-azurerm/vendor/google.golang.org/grpc/server.go:1617 +0xd0c
google.golang.org/grpc.(*Server).serveStreams.func1.2(0xc001096040, 0xc00022a8c0, 0x724a198, 0xc000299500, 0xc000dc06c0)
        /opt/teamcity-agent/work/5d79fe75d4460a2f/src/github.com/terraform-providers/terraform-provider-azurerm/vendor/google.golang.org/grpc/server.go:940 +0xab
created by google.golang.org/grpc.(*Server).serveStreams.func1
        /opt/teamcity-agent/work/5d79fe75d4460a2f/src/github.com/terraform-providers/terraform-provider-azurerm/vendor/google.golang.org/grpc/server.go:938 +0x1fd

Error: The terraform-provider-azurerm_v2.68.0_x5 plugin crashed!

This is always indicative of a bug within the plugin. It would be immensely
helpful if you could report the crash with the plugin's maintainers so that it
can be fixed. The output above should help diagnose the issue.

### Expected Behaviour

resource to be imported into state file and managed with terraform thereafter

### Actual Behaviour

panic: interface conversion: interface {} is nil, not string

### Steps to Reproduce

NOTE: Again x represents actual case and length

<!--- Please list the steps required to reproduce the issue. --->

1. `terraform import azurerm_logic_app_workflow.xxxxxxxxxxxx-xxxxxx-xxxxxxxxx-xxxxxxxx /subscriptions/xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx/resourceGroups/xxx-xxx-xxxxxx/providers/Microsoft.Logic/workflows/Xxxxxxxxxxxx-Xxxxxx_Xxxxxxxxx_Xxxxxxxxx`

--- a/azurerm/internal/services/logic/logic_app_workflow_resource.go
+++ b/azurerm/internal/services/logic/logic_app_workflow_resource.go
@@ -306,8 +306,12 @@ func resourceLogicAppWorkflowRead(d *pluginsdk.ResourceData, meta interface{}) e
 		}
 		if definition := props.Definition; definition != nil {
 			if v, ok := definition.(map[string]interface{}); ok {
-				d.Set("workflow_schema", v["$schema"].(string))
-				d.Set("workflow_version", v["contentVersion"].(string))
+				if v["$schema"] != nil {
+					d.Set("workflow_schema", v["$schema"].(string))
+				}
+				if v["contentVersion"] != nil {
+					d.Set("workflow_version", v["contentVersion"].(string))
+				}
 			}
 		}
 

